README.md

```diff
@@ -415,6 +446,16 @@ This method makes a call to the
 [/{db}/_design/{ddoc}/_view/{view}](http://docs.couchdb.org/en/1.6.1/api/ddoc/views.html#get--db-_design-ddoc-_view-view)
 endpoint, and the builder supports all the relevant options.
 
+Similarly, to query the total age of Persons in the document using the
+`totalAgeView` builder we can do:
+
+```Scala
+totalAgeView.queryWithReduce[Int]
```

Comment:
I think type param `V` in `db.query.view[K,V]` would refer to the value type of the map function, not necessarily the return type of the reduce operation (which theoretically can be diff). 
Technically, I think you can just use the `map` function of a view that has both a `map` and `reduce` defined so we need to retain all three types the `K` and `V` for the map function and the type `T` for the reduce return type.
To use only the map function of a map and reduce view though, we would have to explicitly invoke `reduce(false)` in the builder since `reduce` defaults to `true` if a reduction function is defined in the view. I just noticed however, that we can't do that presently because `reduce(Boolean)` is a private method in our builder, but that is an easy fix